sylvia-derive/src/message.rs

```diff
@@ -297,6 +349,39 @@ impl<'a> EnumMessage<'a> {
         let ep_name = msg_ty.emit_ep_name();
         let messages_fn_name = Ident::new(&format!("{}_messages", ep_name), name.span());
 
+        let is_func_empty = match_arms.clone().into_iter().count() == 0;
+
+        let ctx_creation = if !is_func_empty {
```

Comment:
No context creation if no message is registered. This avoids unknown types for ctx (because the type is derived from the functions it's passed down to)